build/internal/test/test.go

```diff
@@ -0,0 +1,72 @@
+package test
+
+import (
+	"context"
+
+	"dagger.io/dagger"
+)
+
+func Test(ctx context.Context, client *dagger.Client, flipt *dagger.Container) error {
+	// create Redis service container
+	redisSrv := client.Container().
+		From("redis").
+		WithExposedPort(6379).
+		WithExec(nil)
+
+	_, err := flipt.
+		WithServiceBinding("redis", redisSrv).
+		WithEnvVariable("REDIS_HOST", "redis:6379").
+		WithExec([]string{"go", "test", "-tags", "assets", "-p", "1", "./..."}).
```

Comment:
idk if we need `assets` tag here

Reply:
hmm yes I think you're right. As we're not exercising those paths within Flipts unit tests.